lib/zendesk_apps_support/manifest.rb

```diff
@@ -115,7 +117,9 @@ def initialize(manifest_text)
         instance_variable_set(:"@#{ruby}", m[json])
       end
       @requirements_only ||= false
+      @requirements_only = false if @requirements_only.is_a?(Array) && @requirements_only.empty?
```

Comment:
updated description. although maybe i should also support strings too.
I did mention I'm not 100% sure this is the best way to go (i'd almost prefer a "requirements" location), but it's definitely the easiest to implement